tools/AllTools.sln

```diff
@@ -0,0 +1,61 @@
+﻿
```

Comment:
Why do we need tools dedicated solution? We just deleted the one dedicated solution for tutorials. I vote for the single solution entry point - AllBuild.sln

Reply:
Deleted tools dedicated solution